subworkflows/nf-core/vcf_annotate_ensemblvep_snpeff/tests/main.nf.test.snap

```diff
@@ -0,0 +1,338 @@
+{
+    "sarscov2 - snpeff": {
+        "content": [
+            [
+                [
+                    {
+                        "groupSize": 2,
+                        "groupTarget": {
+                            "id": "custom_test",
+                            "single_end": false
+                        }
+                    },
+                    "custom_test.vcf.gz,variantsMD5:e9323877c6352459351537ac6539914a",
+                    "custom_test.vcf.gz.tbi"
+                ],
+                [
+                    {
+                        "groupSize": 2,
+                        "groupTarget": {
+                            "id": "test",
+                            "single_end": false
+                        }
+                    },
+                    "test.vcf.gz,variantsMD5:e9323877c6352459351537ac6539914a",
+                    "test.vcf.gz.tbi"
+                ]
+            ],
+            [
+                
+            ],
+            [
+                [
+                    {
+                        "id": "custom_test0_annotated",
+                        "single_end": false
+                    },
+                    "custom_test0_annotated.csv"
+                ],
+                [
+                    {
+                        "id": "custom_test1_annotated",
+                        "single_end": false
+                    },
+                    "custom_test1_annotated.csv"
```

Comment:
Can't even have the header string for example?

Reply:
Or number of rows using nft-csv?

Reply:
Good idea I forgot that plugin exists :grin: let me have a go at it

Reply:
The CSV file isn't a valid CSV it's some sort merge of several CSV files, which causes the plugin to fail.